### PR TITLE
Fix abstract key error

### LINKED ISSUE #1869: Invite Assignment: fix key error when abstract is not present
Check this process function error:https://api2.openreview.net/logs/process?id=8PVIRQUTK0Udwn

### PR DESCRIPTION
**Pull request by enrubio**
- Resolves #1869

Fixes the email string when it tries to retrieve the abstract in the PC submission revision & invite assignment process functions.

Still need to recreate these errors in the tests:
https://api2.openreview.net/logs/process?id=8PVIRQUTK0Udwn
https://api2.openreview.net/logs/process?id=BiSMuQN2Yk

### CODE REVIEW

**Comment by melisabok** [comment]
I would always start recreating the errors first and then try to fix them :) 

That is the only way to verify the error is reproducible. It seems test_worksho_v2.py removed the abstract from the submission revision invitation: https://github.com/openreview/openreview-py/blob/master/tests/test_workshop_v2.py#L440

I would suggest to remove the abstract field name from the submission and:
1. Post a PC_Revision and reproduce the process function error
2. The venue has a manual assignment so you can set the invite assignment invitation manually and try to invite a reviewer. @celestemartinez please help with the instructions to setup the invite recruitment invitation.

**Review by melisabok — APPROVED** [review]
💯

File: openreview/venue/process/pc_submission_revision_process.py [hunk 1]
```diff
@@ -11,12 +11,14 @@ def process(client, edit, invitation):
 
     subject = f'''{short_name} has received a new revision of your submission titled {submission.content['title']['value']}'''
 
+    abstract_string = f'''
+Abstract {submission.content['abstract']['value']}
```
Comment (celestemartinez) [inline openreview/venue/process/pc_submission_revision_process.py:15]:
```suggestion
Abstract: {submission.content['abstract']['value']}
```

**Review by celestemartinez — APPROVED** [review]
Looks great!

**Pull request merged**